### DISCUSSION TITLE
Post-deploy hook to tell Rollbar about the deployment

### DISCUSSION THREAD

**Discussion by nm2501** [vercel/community#181] [Idea]
Whenever there's a production deployment on Vercel I'd like to have Rollbar kept informed so that it's easier to correlate error spikes in Rollbar with various deployments.

Rollbar has an [API](https://explorer.docs.rollbar.com/#operation/post-deploy) for this, but I can't find any way to hook into the Vercel deployment and invoke the Rollbar API.

There's a Sentry integration for Vercel, but unfortunately no Rollbar integration.

Any suggestions?

**Comment by Pieparker** [comment]
Hey there, we've moved this to the Vercel Community Discussions as a feature suggestion. Thank you!
